a2ml/tasks_queue/data_source_api_pandas.py

```diff
@@ -0,0 +1,1083 @@
+import arff
```

Comment:
Is this entire file necessary to port over? Could we just move over the parts that are necessary? Also, it's not covered by tests in this project.